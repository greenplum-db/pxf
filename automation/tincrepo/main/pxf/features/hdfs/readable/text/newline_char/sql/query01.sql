-- @description query01 for PXF HDFS Readable newline character

SELECT * from pxf_newline_char;