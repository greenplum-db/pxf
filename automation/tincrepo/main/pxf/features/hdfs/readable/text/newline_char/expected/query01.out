-- @description query01 for PXF HDFS Readable newline character
SELECT * from pxf_newline_char;
      name      | num | dub |    longnum    | bool 
----------------+-----+-----+---------------+------
 newline_row_1  |   1 |   1 |  100000000000 | f
 newline_row_2  |   2 |   2 |  200000000000 | t
 newline_row_3  |   3 |   3 |  300000000000 | f
 newline_row_4  |   4 |   4 |  400000000000 | t
 newline_row_5  |   5 |   5 |  500000000000 | f
 newline_row_6  |   6 |   6 |  600000000000 | t
 newline_row_7  |   7 |   7 |  700000000000 | f
 newline_row_8  |   8 |   8 |  800000000000 | t
 newline_row_9  |   9 |   9 |  900000000000 | f
 newline_row_10 |  10 |  10 | 1000000000000 | t
(10 rows)

